spec_finder.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python
+import urllib.request
+import json
+import re
+import difflib
+import os
+import sys
+
+def _demarkdown(t):
+    return t.replace('**', '').replace('`', '').replace('"', '')
+
+def get_spec(force_refresh=False):
+    spec_path = './specification.json'
+    data = ""
+    if os.path.exists(spec_path) and not force_refresh:
+        with open(spec_path) as f:
+            data = ''.join(f.readlines())
+    else:
+        # TODO: Status code check
```

Comment:
Did you want to address this TODO in this PR?

Reply:
No. :)